```
 .::error file=tests/components/roborock/test_select.py,line=57::test_update_success[select.roborock_s7_maxv_mop_mode-deep-set_mop_mode-expected_params0]%0A%0Ahomeassistant.exceptions.ServiceValidationError: Option deep is not valid for entity select.roborock_s7_maxv_mop_mode, valid options are:
F.::error file=tests/components/roborock/test_select.py,line=57::test_update_success[select.roborock_s7_maxv_mop_intensity-mild-set_water_box_custom_mode-expected_params1]%0A%0Ahomeassistant.exceptions.ServiceValidationError: Option mild is not valid for entity select.roborock_s7_maxv_mop_intensity, valid options are:
F.........::error file=tests/components/roborock/test_select.py,line=108::test_update_failure%0A%0AAssertionError: Regex pattern did not match.%0A  Expected regex: 'Error while calling SET_MOP_MOD'%0A  Actual message: 'Option deep is not valid for entity select.roborock_s7_maxv_mop_mode, valid options are: '

```

--- a/homeassistant/components/roborock/strings.json
+++ b/homeassistant/components/roborock/strings.json
@@ -118,9 +118,12 @@
           "max": "Max",
           "medium": "[%key:common::state::medium%]",
           "mild": "Mild",
+          "min": "Min",
           "moderate": "Moderate",
           "off": "[%key:common::state::off%]",
+          "slight": "Slight",
           "smart_mode": "[%key:component::roborock::entity::select::mop_mode::state::smart_mode%]",
+          "standard": "[%key:component::roborock::entity::select::mop_mode::state::standard%]",
           "vac_followed_by_mop": "Vacuum followed by mop"
         }
       },
@@ -448,6 +451,7 @@
               "max_plus": "Max plus",
               "medium": "[%key:common::state::medium%]",
               "off": "[%key:common::state::off%]",
+              "off_raise_main_brush": "Off (raised brush)",
               "quiet": "Quiet",
               "silent": "Silent",
               "smart_mode": "[%key:component::roborock::entity::select::mop_mode::state::smart_mode%]",

--- a/tests/components/roborock/conftest.py
+++ b/tests/components/roborock/conftest.py
@@ -10,7 +10,14 @@ from typing import Any
 from unittest.mock import AsyncMock, Mock, PropertyMock, patch
 
 import pytest
-from roborock import HomeDataRoom, MultiMapsListMapInfo, RoborockCategory
+from roborock import (
+    CleanRoutes,
+    HomeDataRoom,
+    MultiMapsListMapInfo,
+    RoborockCategory,
+    VacuumModes,
+    WaterModes,
+)
 from roborock.data import (
     CombinedMapInfo,
     DnDTimer,
@@ -307,6 +314,20 @@ def create_v1_properties(network_info: NetworkInfo) -> AsyncMock:
         trait_spec=StatusTrait,
         dataclass_template=STATUS,
     )
+    _fan_speed_mapping = {m.code: m.value for m in VacuumModes}
+    _water_mode_mapping = {m.code: m.value for m in WaterModes}
+    _mop_route_mapping = {m.code: m.value for m in CleanRoutes}
+    v1_properties.status.fan_speed_options = list(VacuumModes)
+    v1_properties.status.fan_speed_mapping = _fan_speed_mapping
+    v1_properties.status.fan_speed_name = _fan_speed_mapping.get(STATUS.fan_power)
+    v1_properties.status.water_mode_options = list(WaterModes)
+    v1_properties.status.water_mode_mapping = _water_mode_mapping
+    v1_properties.status.water_mode_name = _water_mode_mapping.get(
+        STATUS.water_box_mode
+    )
+    v1_properties.status.mop_route_options = list(CleanRoutes)
+    v1_properties.status.mop_route_mapping = _mop_route_mapping
+    v1_properties.status.mop_route_name = _mop_route_mapping.get(STATUS.mop_mode)
     v1_properties.dnd = make_dnd_timer(dataclass_template=DND_TIMER)
     v1_properties.clean_summary = make_mock_trait(
         trait_spec=CleanSummaryTrait,

